go/internal/feast/featurestore.go

```diff
@@ -45,18 +50,29 @@ func NewFeatureStore(config *registry.RepoConfig, callback transformation.Transf
 	if err != nil {
 		return nil, err
 	}
-
-	registry, err := registry.NewRegistry(config.GetRegistryConfig(), config.RepoPath)
+	registryConfig, err := config.GetRegistryConfig()
+	if err != nil {
+		return nil, err
+	}
+	registry, err := registry.NewRegistry(registryConfig, config.RepoPath, config.Project)
 	if err != nil {
 		return nil, err
 	}
-	registry.InitializeRegistry()
+	err = registry.InitializeRegistry()
+	if err != nil {
+		return nil, err
+	}
+	sanitizedProjectName := strings.Replace(config.Project, "_", "-", -1)
+	productName := os.Getenv("PRODUCT")
+	endpoint := fmt.Sprintf("%s-transformations.%s.svc.cluster.local:80", sanitizedProjectName, productName)
```

Comment:
Is this an external Transformation server? I guess we need to remove this and related code? @EXPEbdodla

Reply:
Initially transformations are running with GOPY bindings. That's not working at scale. So we implemented it similar to Java Feature Server.

Reply:
Thank you, @EXPEbdodla  I guess I need to work on this part.

Reply:
I chose an "easy" fix for this. :crying_cat_face:

Reply:
Yes. We should get the transformation server endpoint from feature_store.yaml.

Reply:
updated. :)